docs/videos/index.html

```diff
@@ -54,6 +54,13 @@ <h4>Short Introduction to Web of Things</a></h4>
 				<!-- 2019-10-16 -->
 			</a>
         </div>
+        <div class="col-md-6">
```

Comment:
I think this might come as first entry in the "Tutorial" section.
I guess the order is "recent first"

Reply:
I have moved to first place